### PR TITLE
unit tests written

### PR DESCRIPTION
**Pull request by ilyashn**

### CODE REVIEW

**Comment by Arthasasyan** [comment]
@ilyashn please, write [`.travis.yml`](https://docs.travis-ci.com/user/languages/python) file

**Comment by Arthasasyan** [comment]
Note that bot does not require to work on python under 3.5

**Comment by Arthasasyan** [comment]
Please, add new lines at the end of all files

File: config/config.ini [hunk 1]
```diff
@@ -1,2 +1,2 @@
 [Bot]
-token = stub
\ No newline at end of file
+token = NjQzMTAyMDc5MzkyNjc3ODk4.XclMlA.wDkz8rZHXMcy-MNwsuWJw38v7wU
```
Comment (Arthasasyan) [inline config/config.ini]: Don't write bot token publically

**Review by Arthasasyan — CHANGES_REQUESTED** [review]

**Comment by ilyashn** [comment]
[Настройка конфига для Travis CI](https://trello.com/c/OeLjwC5b/20-%D0%BD%D0%B0%D1%81%D1%82%D1%80%D0%BE%D0%B9%D0%BA%D0%B0-%D0%BA%D0%BE%D0%BD%D1%84%D0%B8%D0%B3%D0%B0-%D0%B4%D0%BB%D1%8F-travis-ci)

**Pull request merged**